docs/source/user_guide/timeseries.ipynb

```diff
@@ -0,0 +1,374 @@
+    "pd.testing.assert_series_equal(expected_preds, naive_baseline_preds)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Feature Enginering\n",
+    "\n",
+    "EvalML uses the values of `gap`, `forecast_horizon`, and `max_delay` to calculate a \"window\" of allowed dates that can be used for engineering the features of each row in the validation/test set. The formula for computing the bounds of the window is:\n",
+    "\n",
+    "<br>  \n",
+    "<div align=\"center\"><b>[t - (max_delay + forecast_horizon + gap), t - (forecast_horizon + gap)]</b></div>\n",
+    "\n",
+    "\n",
+    "As an example, this is what the features for the first five days of August would look like in our current problem:\n",
+    "\n",
+    "![features](ts_viz/feature_engineering_window_viz.png)\n",
+    "\n",
+    "The estimator then takes these features to generate predictions:\n",
+    "\n",
+    "![estimator predictions](ts_viz/estimator_viz.png)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Evaluate best pipeline on test data\n",
+    "\n",
+    "Now that we have covered the mechanics of how EvalML runs AutoMLSearch for time series pipelines, we can compare the performance on the test set of the best pipeline found during search and the baseline pipeline."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pl = automl.best_pipeline\n",
+    "\n",
+    "pl.fit(X_train, y_train)\n",
+    "\n",
+    "best_pipeline_score = pl.score(X_test, y_test, ['R2'], X_train, y_train)['R2']"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "best_pipeline_score"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "baseline = automl.get_pipeline(0)\n",
+    "baseline.fit(X_train, y_train)\n",
+    "naive_baseline_score = baseline.score(X_test, y_test, ['R2'], X_train, y_train)['R2']"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "naive_baseline_score"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The pipeline found by AutoMLSearch has a 268% improvement over the naive forecast!"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "automl.objective.calculate_percent_difference(best_pipeline_score, naive_baseline_score)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Visualize the predictions over time"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "data = pd.DataFrame({\"dates\": X_test.Date,\n",
```

Comment:
I will file an issue for updating `graph_predicition_vs_actual_over_time` so that we can use it here

Reply:
Filed #2731